hera_cal/abscal.py

```diff
@@ -2029,7 +2029,8 @@ def match_times(datafile, modelfiles, filetype='uvh5', atol=1e-5):
 
     # shift data relative to model if needed
     data_lsts[data_lsts < model_starts[0]] += 2 * np.pi
-
+    # make sure monotonically increasing.
+    data_lsts[data_lsts < data_lsts[0]] += 2 * np.pi
```

Comment:
Doesn't np.unwrap do what you want here?

Reply:
it does. I've changed this.